src/plugins/badge.coffee

```diff
@@ -0,0 +1,769 @@
+    display: inline-block;
+
+    width: 90px;
+    height: 24px;
+
+    vertical-align: top;
+    float: right;
+
+    padding-top: 5px;
+  }
+
+  #sa-badge-embedded-header #sa-badge-embedded-rating .rating-number {
+    display: inline-block;
+
+    font-family: Verdana, Arial, sans-serif !important;
+    font-weight: bold;
+    font-size: 14px !important;
+    color: #{switch configuration.theme
+            when 'white' then 'black'
+            when 'black' then 'white'};
+  }
+
+  #sa-badge-embedded-header #sa-badge-embedded-rating .stars-container {
+    display: inline-block;
+  }
+
+  #sa-badge-embedded-header #sa-badge-embedded-rating .stars-container .sa-badge-star {
+    display: inline-block;
+    margin-right: -3px;
+
+    width: 11px;
+    height: 10px;
+
+    background-image: url('#{asset_url("badge/embedded/theme/#{configuration.theme}/star_empty.png")}');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 11px 10px;
+  }
+
+  #sa-badge-embedded-header #sa-badge-embedded-rating .stars-container .sa-badge-full-star {
+    background-image: url('#{asset_url("badge/embedded/theme/#{configuration.theme}/star_full.png")}');
+  }
+
+  #sa-badge-embedded-header #sa-badge-embedded-rating .stars-container .sa-badge-half-star {
+    background-image: url('#{asset_url("badge/embedded/theme/#{configuration.theme}/star_half.png")}');
+  }
+
+  #sa-badge-embedded-header #sa-badge-embedded-rating .stars-container .sa-badge-empty-star {
+    background-image: url('#{asset_url("badge/embedded/theme/#{configuration.theme}/star_empty.png")}');
+  }
+
+  #sa-badge-embedded-footer #sa-badge-embedded-more-button {
+    display: block;
+
+    padding: 6px 15px 7px 15px;
+    width: 40%;
+    margin: auto;
+
+    background-color: #f68b24 !important;
+    color: white !important;
+    font-family: Verdana, Arial, sans-serif !important;
+    font-size: 11px !important;
+
+    cursor: pointer;
+    border-radius: 2px;
+
+    text-decoration: none;
+
+    transition: none;
+    transition: background-color .2s ease-in-out;
+  }
+
+  #sa-badge-embedded-footer #sa-badge-embedded-more-button:hover {
+    color: white !important;
+    background-color: #d8721c !important;
+  }
+
+  @media only screen and (-webkit-min-device-pixel-ratio: 2),
+         only screen and (min--moz-device-pixel-ratio: 2),
+         only screen and (  -o-min-device-pixel-ratio: 2/1),
+         only screen and (     min-device-pixel-ratio: 2),
+         only screen and (     min-resolution: 192dpi),
+         only screen and (     min-resolution: 2dppx) {
+
+    #sa-badge-embedded-header #sa-badge-embedded-logo {
+      background-image: url('#{asset_url("badge/embedded/theme/#{configuration.theme}/logo_@@flavor@2x.png")}');
+    }
+
+    #sa-badge-embedded-header #sa-badge-embedded-rating .stars-container .sa-badge-star {
+      background-image: url('#{asset_url("badge/embedded/theme/#{configuration.theme}/star_empty@2x.png")}');
+  }
```

Comment:
Indentation, also i think that this one seems like a duplicate
check below: `#sa-badge-embedded-header #sa-badge-embedded-rating .stars-container .sa-badge-empty-star `